packages/nulp_elite/public/locales/be/translation.json

```diff
@@ -326,7 +326,7 @@
   "REQUEST_TO_CREATE": "তৈরি করার অনুরোধ",
```

Comment:
@suraj-tekdi follow the std format for MR Title. please check this [document](https://docs.google.com/document/d/17DCzkIxY1ESvYYmQGyK1q8Wa48fPgUZMVvgfynWeF68/edit?tab=t.0#heading=h.23ywrfmjdubq)